src/Microsoft.OpenApi.OData.Reader/PathItem/PathItemHandler.cs

```diff
@@ -101,5 +113,54 @@ protected virtual void AddOperation(OpenApiPathItem item, OperationType operatio
             IOperationHandler operationHander = provider.GetHandler(Path.Kind, operationType);
             item.AddOperation(operationType, operationHander.CreateOperation(Context, Path));
         }
+
+        /// <summary>
+        /// Set the parameters information for <see cref="OpenApiPathItem"/>
+        /// </summary>
+        /// <param name="item">The <see cref="OpenApiPathItem"/>.</param>
+        protected virtual void SetParameters(OpenApiPathItem item)
+        {
+            foreach (ODataKeySegment keySegment in Path.OfType<ODataKeySegment>())
+            {
+                IDictionary<string, string> mapping = ParameterMappings[keySegment];
+                foreach (var parameter in Context.CreateKeyParameters(keySegment, mapping))
+                {
+                    AppendParameter(item, parameter);
+                }
+            }
+
+            // Add the route prefix parameter v1{data}
+            if (Context.Settings.RoutePathPrefixProvider != null && Context.Settings.RoutePathPrefixProvider.Parameters != null)
```

Comment:
```suggestion
            if (Context.Settings.RoutePathPrefixProvider?.Parameters != null)
```

Reply:
Good suggestion. I have adopted it. Thanks.